app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,118 @@
+html,
+body{
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  color: #fff;
+  -webkit-font-smoothing: antialiased;
+  -moz-font-smoothing: antialiased;
+  -o-font-smoothing: antialiased;
+}
+
+@font-face {
+    font-family: "Guardian Egyptian Headline";
+    src: url('http://interactive.guim.co.uk/fonts/pasteup.guim.co.uk/fonts/latin1/Guardian-Egyp-Web-Regular.eot');
```

Comment:
It looks like interactive.guim.co.uk also supports HTTPS, so it would be better to use `//` rather than hardcoding to `http://` for all these links. Probably doesn't need to be fixed for the trial though, as these embeds will be in articles, which are not yet on HTTPS.